ProjectShop/WishList/models.py

```diff
@@ -0,0 +1,8 @@
+from django.db import models
+
+from UserApp.models import User
+
+
+class WishList(models.Model):
+    user_id = models.ForeignKey(User, on_delete=models.CASCADE)
+    # product_id = models.ForeignKey(Product, on_delete=models.CASCADE)
```

Comment:
We'd better use many to many relationship to Product, then we could include more than one product inside wishlist.